src/test/app/claims/models/claim.ts

```diff
@@ -453,9 +459,30 @@ describe('Claim', () => {
       claim.settlement = new Settlement()
       claim.settlementReachedAt = moment()
 
-      expect(claim.stateHistory).to.have.lengthOf(2)
-      expect(claim.stateHistory[1].status).to.equal(ClaimStatus.OFFER_SUBMITTED)
+      expect(claim.stateHistory).to.have.lengthOf(3)
       expect(claim.stateHistory[0].status).to.equal(ClaimStatus.OFFER_SETTLEMENT_REACHED)
+      expect(claim.stateHistory[1].status).to.equal(ClaimStatus.OFFER_SUBMITTED)
+      expect(claim.stateHistory[2].status).to.equal(ClaimStatus.PAID_IN_FULL_ELIGIBLE)
+    })
+  })
+
+  describe('paidInFullCCJPaidWithinMonth', () => {
+    let claim
+
+    beforeEach(() => {
+      claim = new Claim()
+      claim.moneyReceivedOn = MomentFactory.currentDate()
+    })
+
+    it('should return true when CCJ is paid within month of countyCourtJudgmentRequestedAt', () => {
+      claim.countyCourtJudgmentRequestedAt = MomentFactory.currentDate().add(1, 'month')
+      expect(claim.isCCJPaidWithinMonth()).to.be.equal(true)
```

Comment:
minor, you can use `.to.be.true` and `.to.be.false`

Reply:
Lint gives me the error 

> unused expression, expected an assignment or function call

 when I use this syntax. Any way around this?